### PR TITLE
chore: update and restructure frontend learning resources

### PR DESCRIPTION
**Pull request by heyOnuoha**
- Added new sections: Browser DevTools, PWAs, Web Security, Accessibility, and Deployment Platforms
- Expanded existing categories with additional curated links
- Improved organization and readability of resource documentation
- Ensured all links and sections align with frontend development best practices

### CODE REVIEW

**Comment by heyOnuoha** [comment]
Hello @sadanandpai! Please take a look at my PR, I've added more resources and categories 🔥

File: README.md [hunk 1]
```diff
@@ -165,13 +165,59 @@
 <strong>Web Security</strong>
 
 - 🎥&nbsp;&nbsp;[Web Security](https://frontendmasters.com/courses/web-security/)
+- 📗&nbsp;&nbsp;[OWASP Top Ten Security Risks](https://owasp.org/www-project-top-ten/)
+- 📗&nbsp;&nbsp;[Google Web Fundamentals: Security](https://web.dev/security/)
+- 📗&nbsp;&nbsp;[MDN Web Docs: Security](https://developer.mozilla.org/en-US/docs/Web/Security)
+- 🎥&nbsp;&nbsp;[Cross-Site Scripting and CSRF Prevention](https://www.youtube.com/watch?v=m0EHlfTgGUU)
```
Comment (sadanandpai) [inline README.md]: We can remove 
- OWASP Top Ten Security Risks
- Cross-Site Scripting and CSRF Prevention
as they are not a complete tut or guide.

File: README.md [hunk 2]
```diff
@@ -165,13 +165,59 @@
...
 - 🎥&nbsp;&nbsp;[Develop Accessible Web Apps with React](https://egghead.io/courses/develop-accessible-web-apps-with-react)
+- 📗&nbsp;&nbsp;[Web Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/)
+
+<br>
+
+<strong>Browser DevTools</strong>
+
+- 📗&nbsp;&nbsp;[Chrome DevTools](https://developer.chrome.com/docs/devtools/)
+- 📗&nbsp;&nbsp;[Firefox Developer Tools](https://developer.mozilla.org/en-US/docs/Tools)
+- 🎥&nbsp;&nbsp;[DevTools Crash Course](https://www.youtube.com/watch?v=e1gAyQuIFQo)
```
Comment (sadanandpai) [inline README.md]: Please move this section to the bottom

File: README.md [hunk 3]
```diff
@@ -165,13 +165,59 @@
...
+
+- 📗&nbsp;&nbsp;[MDN Guide to PWAs](https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps)
+- 📗&nbsp;&nbsp;[Google Developers: PWAs](https://web.dev/progressive-web-apps/)
+- 🎥&nbsp;&nbsp;[PWA Crash Course](https://www.youtube.com/watch?v=4XT23X0Fjfk&list=PL4cUxeGkcC9gTxqJBcDmoi5Q2pzDusSL7)
+
+<br>
+
+<strong>Performance Optimization</strong>
+
+- 📗&nbsp;&nbsp;[Core Web Vitals](https://web.dev/vitals/)
```
Comment (sadanandpai) [inline README.md]: Plz move it under Web perf

File: README.md [hunk 4]
```diff
@@ -313,6 +359,9 @@
 - 🚉&nbsp;&nbsp;[Leetcode](https://leetcode.com/)
 - 🚉&nbsp;&nbsp;[Hackerrank](https://www.hackerrank.com/)
 - 🚉&nbsp;&nbsp;[Interviewbit](https://www.interviewbit.com/practice/)
+- 🚉&nbsp;&nbsp;[ICodeThis](https://icodethis.com/)
+- 🚉&nbsp;&nbsp;[React Challenges](https://reactchallenges.live/)
+- 🚉&nbsp;&nbsp;[FrontendPro](https://www.frontendpro.dev/)
```
Comment (sadanandpai) [inline README.md]: React Challenges & FrontendPro are already there

**Review by sadanandpai — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (heyOnuoha) [reply README.md]: Understood! This has been removed

File: README.md (hunk 2 above)
Comment (heyOnuoha) [reply README.md]: Done!

File: README.md (hunk 3 above)
Comment (heyOnuoha) [reply README.md]: Done! 🔥

File: README.md (hunk 4 above)
Comment (heyOnuoha) [reply README.md]: I just saw when you stated this, I somehow missed that haha. They've been removed! Thank you!!!!!!!

**Review by sadanandpai — APPROVED** [review]

**Pull request merged**